services/cron/cron.py

```diff
@@ -78,10 +80,18 @@ def cache_to_file(self):
             log(f'Executing {script}')
             tc.reset(key)
             try:
-                subprocess.Popen(
+                proc = subprocess.Popen(
                     script, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
                 )
+                output, _ = proc.communicate()
             except Exception as e:
                 log(f'Error executing {script}: {e}')
+                DBSession().add(CronJobRun(script=script, exit_status=1))
```

Comment:
Add the exception string here as the proc_output?

Reply:
Done - renamed proc_output to output, and just store the exception string there in that case